packages/compat/webpack/package.json

```diff
@@ -45,7 +45,7 @@
     "typescript": "^5.6.3"
   },
   "peerDependencies": {
-    "@rsbuild/core": "^1.1.2"
+    "@rsbuild/core": "^1.1.4"
```

Comment:
why do we need to bump this version

Reply:
normalize stats with `getStatsOptions` is supported in rsbuild 1.1.3.
https://github.com/web-infra-dev/rsbuild/pull/3987